meerk40t/core/svg_io.py

```diff
@@ -1049,8 +1050,8 @@ def is_child(candidate, parent_node):
                         lock=_lock,
                     )
                     e_list.append(node)
-            except OSError:
-                pass
+            except OSError as e:
+                raise e
```

Comment:
Isn't this the same as not doing an except?

Reply:
Yes. This like was debug. I needed to see what e actually said as part of the debugging.

Reply:
No - my point is that trapping and then raising the same exception is EXACTLY the same as not trapping it at all.

Reply:
I did that to look at the current value of `e` which let me see what the thrown error was. Which is slightly different. But, yes, you are correct. I will remove that change.

Reply:
This and all changes to svg_io were reverted.